### ISSUE TITLE
How to train YOLO_NAS?

### ISSUE THREAD

**Issue by ToniButland1998** [Deci-AI/super-gradients#1043]
### 💡 Your Question

How to train YOLO_NAS on datasets, like coco?

**Comment by johnGettings** [comment]
1) Go to the YOLONAS.md, link on the homepage. There is documentation and even a google colab for finetuning.
2) YOLO_NAS is already trained on coco
